tests/sql/moz-fx-data-shared-prod/search_derived/mobile_search_clients_last_seen_v1/test_day_bit_shifting/expect.yaml

```diff
@@ -23,18 +23,18 @@
   total_searches: 0
   tagged_searches: 0
   engine_searches:
-  - value:
-      total_searches: [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
-      tagged_searches: [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
-      search_with_ads: [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
-      ad_click: [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
+    - value:
+        total_searches: [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
+        tagged_searches: [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
+        search_with_ads: [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
+        ad_click: [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
   days_seen_bytes: AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABw==
   days_searched_bytes: AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABA==
   days_tagged_searched_bytes: AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABA==
   days_searched_with_ads_bytes: AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABA==
   days_clicked_ads_bytes: AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABA==
   days_created_profile_bytes: AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABkA==
-- <<: *base
```

Comment:
This should be a list.

Reply:
I think this may be indented incorrectly, and the row is being read as a a list with 1 item instead of a list with 2 items.